docs/README.md

```diff
@@ -60,27 +60,23 @@ The following environment variables need to be set for running the end to end te
   - Set this to point to your AWS credentials file
 - KUBE_SSH_KEY_PATH
   - The ssh key used to bring up the VM
-- KUBECONFIG
-  - The kubeconfig of the OpenShift cluster
+- WMCB_IMAGE
+  - Registry url for remote WMCB image that needs to be tested. eg. quay.io/<USERNAME>/<IMAGE>:<TAG>
+
+To build the WMCB image, execute:
+```
+podman build -f Dockerfile.tools -t quay.io/<USERNAME>/<IMAGE>:<TAG> .
+``` 
+The WMCB image needs to be pushed to a remote repository:
+```
+podman push quay.io/<USERNAME>/<IMAGE>:<TAG>
+```
 
 Once the above variables are set, you can run the unit and end to end tests by executing:
 ```shell script
 $ hack/run-wmcb-ci-e2e-test.sh
 ```
 
-The hack script can be given the following options:
-- `-v` option takes a list of VM credentials in the order of `instance-id,ip-address,password`. The username defaults
-   to `Administrator`. This allows you to run the tests against existing set of VMs.
-   ```shell script
-   $ hack/run-wmcb-ci-e2e-test.sh -v"aws-instance-id-1,3.135.234.23,password,aws-instance-id-2,3.135.234.23,password"
-   ```
-
-- `-s` option allows you to skip the framework setup. The assumption here is that the framework setup has already been
-  run on the VM.
-  ```shell script
-  $ hack/run-wmcb-ci-e2e-test.sh -v"aws-instance-id,1.2.34.23,password" -s
-  ```
```

Comment:
Is this no longer needed or skippable?

Reply:
We can no longer accept credentials in the form of username or password. So thats no more needed. 
as far as skipping framework setup goes, we can do that through `job.yaml`, and in that case, we will be using the mounted private key to access the VM.

Reply:
I was asking about skipping the framework setup. How are we going to do this through `job.yaml`? I am assuming we have to do it manually. Moreover, can we still reuse an existing `Machine`? I think not so this point becomes moot.

Reply:
Please correct me if there is a way to use an existing VM / `Machine`.

Reply:
actually, if the required machine has all the labels that we add, and if it is in provisioned state. We can actually use that machine to run tests.. 
after i removed skipVMSetup option, every time i had to run the tests the wait would be atleast 12 min. 
we can create a machineset and machine before hand, and we can just delete existing VM attached as a node and reuse the new VM created. This will reduce the wait time to just 1 minute. 
Will bring this up in stamdup

Reply:
added detailed information to Readme.